common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -97,6 +97,15 @@ class ProctoringFields(object):
         scope=Scope.settings,
     )
 
+    hide_after_due = Boolean(
+        display_name=_("Hide Exam Results After Due Date"),
+        help=_(
+            "This setting overrides the default behavior of showing exam results after the due date has passed."
```

Comment:
👍 sounds good to me, I'll get it changed. I'm not 100% sure where this is visible, it's a property field on an xmodule. I'll do some poking around and get back to you